roles/openshift_facts/library/openshift_facts.py

```diff
@@ -470,6 +470,8 @@ def set_aggregate_facts(facts):
             all_hostnames.add(kube_svc_ip)
             internal_hostnames.add(kube_svc_ip)
 
+            facts['node'] = {}
```

Comment:
What does this fix specifically?  A test for `openshift.node` later?

Reply:
It ensures masters get `openshift.node.nodename` fact set for autoapprove. This is required for node autoapproval, not sure if necessary for others though